src/www/ui/api/documentation/openapi.yaml

```diff
@@ -867,6 +867,119 @@ paths:
         default:
           $ref: '#/components/responses/defaultResponse'
 
+  /uploads/{id}/item/{itemId}/tree/view:
+    parameters:
+      - name: id
+        required: true
+        description: Id of the upload
+        in: path
+        schema:
+          type: integer
+      - name: itemId
+        required: true
+        description: Id of the itemId
+        in: path
+        schema:
+          type: integer
+      - name: agentId
+        required: false
+        description: Id of the agent
+        in: query
+        schema:
+          type: integer
+      - name: tagId
+        required: false
+        description: Id of the tag
+        in: query
+        schema:
+          type: integer
+      - name: scanLicenseFilter
+        required: false
+        description: Id of the selected scan license filter
+        in: query
+        schema:
+          type: integer
+      - name: editedLicenseFilter
+        required: false
+        description: Id of the selected edited license filter
+        in: query
+        schema:
+          type: integer
+      - name: flatten
+        required: false
+        description: Flatten the tree
+        in: query
+        schema:
+          type: boolean
+      - name: sort
+        required: false
+        description: Sort the tree
+        in: query
+        schema:
+          type: string
+          enum:
+            - asc
+            - desc
+      - name: search
+        required: false
+        description: Filter the list according to the search string
+        in: query
+        schema:
+          type: string
+      - name: show
+        required: false
+        description: Show all the items of the current container
+        in: query
+        schema:
+          type: boolean
+      - name: openCBoxFilter
+        required: false
+        description: openCBox Filter is checked
+        in: query
+        schema:
+          type: boolean
+      - name: page
+        required: false
+        description: Start page number
+        in: query
+        schema:
+          type: integer
+      - name: limit
```

Comment:
Can keep the `page` and `limit` in the headers to be consistent with other endpoints.